packages/common/src/components/CondaPkgList.tsx

```diff
@@ -143,6 +147,23 @@ export class CondaPkgList extends React.Component<IPkgListProps> {
       return <span>{rowData.name}</span>;
     };
 
+    const versionRender = ({ rowData }: ICellRender): JSX.Element => (
```

Comment:
What happen for not installed packages?

Reply:
Nothing. I think we should open this graph from another place, not from the version of the package but I'm not sure where to add it.

Reply:
What do you think of this mock-up?

The idea is to add an icon in the toolbar above the packages list (it will be absent if mamba is not available - or disabled with a title advertising mamba :wink: ).
On click on the button, this would open the dependency panel (not sure if on the right of the package list is a good place or if it needs to be a separated panel to allow positioning and resize by the user).

Then selecting a package will result in the display of its dependencies graph. If not package selected, a help message such _Select a package to display its dependency graph_.

This means also to change the current behavior as onClick on a row changes the status of the package. This should then be restricted to the status icon probably.

![image](https://user-images.githubusercontent.com/8435071/99505823-9d35c980-2981-11eb-8c98-f9e42aab6d01.png)

Reply:
Sounds good for me!
For Mamba, I would add a title in the toolbar when Mamba is not installed saying something like: "Install mamba to have all the functionalities" or maybe and alert icon to have a smaller icon in the toolbar.